src/tests/test_magic_plot.py

```diff
@@ -106,10 +138,19 @@ def test_validate_arguments(tmp_empty, ip, cell, error_type, error_message):
         "boxplot-with",
         "boxplot-shortcuts",
         "boxplot-nas",
+        "bar-1-col",
+        "bar-1-col-horizontal",
+        "bar-2-col",
+        "pie-1-col",
+        "pie-2-col",
         "boxplot-column-name-with-spaces",
         "histogram-column-name-with-spaces",
+        "bar-column-name-with-spaces",
+        "pie-column-name-with-spaces",
         "boxplot-table-name-with-spaces",
         "histogram-table-name-with-spaces",
+        "bar-table-name-with-spaces",
+        "pie-table-name-with-spaces",
     ],
 )
 def test_sqlplot(tmp_empty, ip, cell):
```

Comment:
I see we have some functionality that we are not testing, such as the colors and cmap. To test it, we can create a baseline_images and use `from matplotlib.testing.decorators import image_comparison, _cleanup_cm`. Please see t[his for example](https://github.com/ploomber/jupysql/blob/b0c30bf1020a73c0a30d9362c598d99d7ac5a0fe/src/tests/test_ggplot.py#L98).

Reply:
I can add it to the PR if u want

Reply:
good point. we're not testing *any* of the plots generated by `%sqlplot`

@mehtamohit013: please add tests for bar and pie, I'll open an issue to test the others.

we're doing testing for the ggplot API so you can use those tests as reference